deeplearning/ml4pl/graphs/programl.proto

```diff
@@ -62,7 +62,7 @@ message Node {
   // contents of a statement, or the name of an identifier.
   required string text = 2;
   // The pre-processed text of a node. This is the
-  required string preprocessed_text = 3;
+  optional string preprocessed_text = 3;
```

Comment:
Mh, okay, it seems the protos already have added capabilities and are even closer to feature parity with the old nx graphs.

Reply:
The nx graphs and protos store the same data, the only real difference is that protos are a little more compact, and an interop with C++ code. The conversion from proto -> nx is defined in:

https://github.com/ChrisCummins/phd/blob/master/deeplearning/ml4pl/graphs/programl.py#L159-L225